include/sys/vnode.h

```diff
@@ -24,7 +24,7 @@ typedef struct componentname componentname_t;
 #define VNOVAL (-1)
 
 /* vnode access modes */
-typedef enum { VEXEC = 1, VWRITE = 2, VREAD = 4 } accmode_t;
+typedef enum { VEXEC = 1, VWRITE = 2, VREAD = 4, VADMIN = 8 } accmode_t;
```

Comment:
Could you add a short comment describing that new mode? Something simple like "being the file owner" should be more explanatory than `VADMIN` alone.

Reply:
Done.